spec/dependent_validations_spec.rb

```diff
@@ -0,0 +1,125 @@
+require 'spec_helper'
+require '../lib/mutations/dependent_validations'
+
+describe "Mutations::DependentValidations" do
+
+  describe "invalid graphs" do
+    class CyclicCommand
+      include Mutations::DependentValidations
+
+      def self.validations
+        {
+          independent_validation: [],
+          cyclic_validation_1: %i[cyclic_validation_2],
+          cyclic_validation_2: %i[cyclic_validation_1],
+        }
+      end
+
+      def independent_validation; end
```

Comment:
Only defining this validation because we detect the cycle in the others before we try to run them.